docs/user-guide/recipes.rst

```diff
@@ -104,6 +104,76 @@ a day.
   Schedule successfully created
 
 
+Uploading Content
+=================
+
+It is easy to upload RPMs, SRPMs and DRPMs from your machine via ``pulp-admin rpm repo uploads <content_type>`` to pulp.
+
+::
+
+  $ pulp-admin rpm repo uploads rpm --repo-id zoo --dir . --dir /tmp/downloaded_rpms --file ../bear-4.1-1.noarch.rpm
+  +----------------------------------------------------------------------+
+                                Unit Upload
+  +----------------------------------------------------------------------+
+
+  Extracting necessary metadata for each request...
+  [==================================================] 100%
+  Analyzing: cow-2.2-3.noarch.rpm
+  ... completed
+
+  Creating upload requests on the server...
+  [==================================================] 100%
+  Initializing: cow-2.2-3.noarch.rpm
+  ... completed
+
+  Starting upload of selected units. If this process is stopped through ctrl+c,
+  the uploads will be paused and may be resumed later using the resume command or
+  canceled entirely using the cancel command.
+
+  Uploading: bear-4.1-1.noarch.rpm
+  [==================================================] 100%
+  2438/2438 bytes
+  ... completed
+
+  Importing into the repository...
+  This command may be exited via ctrl+c without affecting the request.
+
+
+  [\]
+  Running...
+
+  Task Succeeded
+
+
+  Deleting the upload request...
+  ... completed
+
+  Uploading: cow-2.2-3.noarch.rpm
+  [==================================================] 100%
+  2420/2420 bytes
+  ... completed
+
+  Importing into the repository...
+  This command may be exited via ctrl+c without affecting the request.
+
+
+  [\]
+  Running...
+
+  Task Succeeded
+
+
+  Deleting the upload request...
+  ... completed
+
+This command will upload all RPMs in ``/tmp/downloaded_rpms`` directory and an rpm package on
+path ``../bear-4.1-1.noarch.rpm``.  You can specify as many directories and files as you want.
+
+It's also possible to use ``--skip-existing`` flag to skip uploading packages which already are on the server.
+
+.. note::
+  RPM-only deltaRPMs are **not** supported. When you try to upload one it will lead to task failure.
+
```

Comment:
maybe provide couple of more words about what is an rpm-only drpms and what is the difference between them and regular drpms